[2022-07-01 15:23:09,613] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: ingest_data.save_data_task scheduled__2022-06-30T21:28:00+00:00 [queued]>
[2022-07-01 15:23:09,617] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: ingest_data.save_data_task scheduled__2022-06-30T21:28:00+00:00 [queued]>
[2022-07-01 15:23:09,617] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-07-01 15:23:09,617] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-07-01 15:23:09,617] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-07-01 15:23:09,633] {taskinstance.py:1377} INFO - Executing <Task(_PythonDecoratedOperator): save_data_task> on 2022-06-30 21:28:00+00:00
[2022-07-01 15:23:09,636] {standard_task_runner.py:52} INFO - Started process 19110 to run task
[2022-07-01 15:23:09,637] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'ingest_data', 'save_data_task', 'scheduled__2022-06-30T21:28:00+00:00', '--job-id', '1186', '--raw', '--subdir', 'DAGS_FOLDER/CarPriceDag.py', '--cfg-path', '/tmp/tmpcadtwo9o', '--error-file', '/tmp/tmpfnhrjacr']
[2022-07-01 15:23:09,638] {standard_task_runner.py:80} INFO - Job 1186: Subtask save_data_task
[2022-07-01 15:23:09,668] {task_command.py:369} INFO - Running <TaskInstance: ingest_data.save_data_task scheduled__2022-06-30T21:28:00+00:00 [running]> on host DESKTOP-9B44JSS.localdomain
[2022-07-01 15:23:09,740] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=ingest_data
AIRFLOW_CTX_TASK_ID=save_data_task
AIRFLOW_CTX_EXECUTION_DATE=2022-06-30T21:28:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-30T21:28:00+00:00
[2022-07-01 15:23:09,886] {python.py:173} INFO - Done. Returned value was: None
[2022-07-01 15:23:09,891] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=ingest_data, task_id=save_data_task, execution_date=20220630T212800, start_date=20220701T132309, end_date=20220701T132309
[2022-07-01 15:23:09,932] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-01 15:23:09,954] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
